lib/lotus/utils/kernel.rb

```diff
@@ -418,8 +426,9 @@ def self.Integer(arg)
       #   # Missing #respond_to?
       #   input = BasicObject.new
       #   Lotus::Utils::Kernel.BigDecimal(input) # => TypeError
-      def self.BigDecimal(arg)
+      def self.BigDecimal(arg, precision: DEFAULT_PRECISION)
         case arg
+        when ->(a) { a.respond_to?(:to_d) && a.class == Rational } then arg.to_d(precision)
```

Comment:
@jodosha take a look at this line. We need add precision for BigDecimal when you are using Rational number, because only a Rational number needs an argument for `#to_d` method I added this line, what do you think? I don't like so much this solution.